lib/features/autentication/login_page.dart

```diff
@@ -0,0 +1,86 @@
+import 'package:flutter/material.dart';
+import 'package:firebase_auth/firebase_auth.dart';
+import 'package:moneyapp/router/go_router.dart';
+
+class LoginPage extends StatefulWidget {
+  const LoginPage({super.key});
+
+  @override
+  LoginPageState createState() => LoginPageState();
+}
+
+class LoginPageState extends State<LoginPage> {
+  final TextEditingController _emailController = TextEditingController();
+  final TextEditingController _passwordController = TextEditingController();
+
+  final GlobalKey<ScaffoldMessengerState> _scaffoldMessengerKey =
```

Comment:
We are working with BLoC+Hooks state management. Don't use Flutters default state management (StatefulWidget). Use HookWidget instead, and bind controllers in the build method, using `useTextEditingController()`. For snackbar, use `ScaffoldMessenger.of(context).showSnackBar(snackBar);`